common/install_miopen.sh

```diff
@@ -59,7 +59,8 @@ MIOPEN_CMAKE_COMMON_FLAGS="
 "
 # Pull MIOpen repo and set DMIOPEN_EMBED_DB based on ROCm version
 if [[ $ROCM_INT -ge 50700 ]] && [[ $ROCM_INT -lt 50800 ]]; then
-    MIOPEN_BRANCH="release/rocm-rel-5.7-staging"
+    echo "ROCm 5.7 MIOpen does not need any patches, do not build from source"
+    exit 0
```

Comment:
@jeffdaily We needed to conditionalize the removal of miopen-hip package in https://github.com/pytorch/builder/pull/1541/commits/25592a79c2a00325b622c5cac47c244d83753158#diff-9c1e7d70fdf39f80006993ef539e80e8f409a80949b1e80a061bd54ef745adcaR36, otherwise ROCm5.7 manylinux images end up having no miopen installed. :(
This showed up in the CI jobs for https://hud.pytorch.org/pytorch/pytorch/pull/109571?sha=c37636c16ade375e48e38c0cb790f5a75db20a81

Reply:
Fix attempted in https://github.com/pytorch/builder/pull/1544